sdv/metadata/anonymization.py

```diff
@@ -74,26 +75,27 @@ def _detect_provider_name(function_name):
         return '.'.join(module[2:])
 
 
-def get_anonymized_transformer(function_name, function_kwargs=None):
+def get_anonymized_transformer(function_name, function_kwargs=None, locales=None):
     """Get an instance with an ``AnonymizedFaker`` for the given ``function_name``.
 
     Args:
         function_name (str):
             String representing predefined ``sdtype`` or a ``faker`` function.
-
         function_kwargs (dict):
             Keyword args to pass into the ``function_name``  when being called if needed.
             This is optional.
+        locales (str or list):
+            The locale(s) to use instead of the default locale. Optional.
     """
     function_kwargs = function_kwargs or {}
     if function_name in SDTYPE_ANONYMIZERS:
         function_kwargs.update(SDTYPE_ANONYMIZERS[function_name])
-        return AnonymizedFaker(**function_kwargs)
+        return AnonymizedFaker(locales=locales, **function_kwargs)
 
-    provider_name = _detect_provider_name(function_name)
+    provider_name = _detect_provider_name(function_name, locales=locales)
     function_kwargs.update({
         'function_name': function_name,
         'provider_name': provider_name
     })
```

Comment:
Overall we can just add `locales` to `function_kwargs` and pass it down the line:
```
function_kwargs.update({
    'locales': locales,
    ...
})
```

Reply:
Looking at it again, I guess I'm a little confused on what `function_kwargs` is actually intended to be. `AnonymizedFaker` takes an argument called `function_kwargs`, which are keyword arguments for the Faker function itself. [This test](https://github.com/sdv-dev/SDV/blob/7194baa81668dc6f56dd0fd1627377b392be9310/tests/unit/metadata/test_anonymization.py#L31) seems to indicate that was the intended purpose of the argument here as well, but in practice we're using it for keyword arguments for `AnonymizedFaker` itself. Actually running this test without mocking `AnonymizedFaker` errors, since it doesn't accept the `domain` keyword argument.

Reply:
I think renaming the variable and using it for init kwargs should help, but I'm not sure if we want to add/fix the ability to pass keyword arguments directly to the Faker function through the `function_kwargs` param in `AnonymizedFaker`